aten/src/ATen/core/MetaFallbackKernel.cpp

```diff
@@ -0,0 +1,33 @@
+#include <torch/library.h>
+#include <ATen/core/dispatch/Dispatcher.h>
+#include <c10/core/impl/PyInterpreter.h>
+
+namespace at {
+namespace impl {
+
+static void metaFallback(
+    const c10::OperatorHandle& op,
+    c10::DispatchKeySet dispatch_keys,
+    torch::jit::Stack* stack) {
+  bool resolved = c10::Dispatcher::singleton().maybeImportAbstractImplPyStub(op.operator_name());
```

Comment:
What's the relationship now between this and the `torch._custom_op.impl.global_registry`? IIUC, it seems like fake tensor will check both, but if you run vanilla meta tensor logic, we will ignore the global_registry and only try this path.

Reply:
Oh nvm - that call will just do the python import, and the user's python module will (ideally) call `.abstract_impl`, populating both the global registry and registering a meta kernel.

I wasn't sure at first whether calling a custom op on the meta device would repeatedly hit this fallback, but that's wrong - the import will populate the dispatcher with a kernel registered directly to the meta key, which will now have higher precedence than the boxed fallback, so redispatching will do the right thing.

Reply:
What happens if someone uses `impl_abstract_pystub`, but their python module doesn't properly register a meta kernel to the dispatcher? Will they infinite loop in this fallback if you try to call their op with meta tensors?